### DISCUSSION TITLE
Feature sequence not showing

### DISCUSSION THREAD

**Discussion by anamariaelek** [GMOD/jbrowse-components#3511] [Q&A]
I have tracks with peaks in gff file added to two of my genome browser. In one assembly, it is possible to get feature sequence (as described [here](https://jbrowse.org/jb2/docs/user_guides/feature_sequence/#other-feature-types)), while in the other this option is not available, shown below: 

1) feature sequence shown
gff:
```
##gff-version 3
##source-version rtracklayer 1.56.1
##date 2023-02-01
1	rtracklayer	sequence_feature	10353	10601	.	.	.	name=peak1
1	rtracklayer	sequence_feature	237628	237876	.	.	.	name=peak2
1	rtracklayer	sequence_feature	521460	521708	.	.	.	name=peak3
1	rtracklayer	sequence_feature	540844	541092	.	.	.	name=peak4
1	rtracklayer	sequence_feature	559291	559539	.	.	.	name=peak5
1	rtracklayer	sequence_feature	565199	565447	.	.	.	name=peak6
1	rtracklayer	sequence_feature	566632	566880	.	.	.	name=peak7
```
feature:
![image](https://user-images.githubusercontent.com/20280755/217586342-486640b4-0539-41a3-901c-6a3e909444fa.png)

2) feature sequence not shown:
gff:
```
##gff-version 3
##source-version rtracklayer 1.56.1
##date 2023-02-08
NC_064034.1	rtracklayer	sequence_feature	3218	3466	.	.	.	name=peak1
NC_064034.1	rtracklayer	sequence_feature	4746	4994	.	.	.	name=peak2
NC_064034.1	rtracklayer	sequence_feature	5890	6138	.	.	.	name=peak3
NC_064034.1	rtracklayer	sequence_feature	26097	26345	.	.	.	name=peak4
NC_064034.1	rtracklayer	sequence_feature	161936	162184	.	.	.	name=peak5
NC_064034.1	rtracklayer	sequence_feature	225008	225256	.	.	.	name=peak6
NC_064034.1	rtracklayer	sequence_feature	242597	242845	.	.	.	name=peak7
```
feature:
![image](https://user-images.githubusercontent.com/20280755/217586136-4996f868-882a-4a12-a468-b1cb9963e586.png)

I don't know where this difference might be coming from, any ideas appreciated.

Thanks,
Anamaria

**Comment by cmdcolin** [comment]
do you happen to know what version you are using? I can't see any condition where it is conditionally shown in the latest version

**Comment by anamariaelek** [comment]
Ah, that could be it, in the first case it's 2.3.3, and in the second one it's older, 2.1.0
